src/ipelib/flip/RSPRIM.f

```diff
@@ -276,6 +275,8 @@ SUBROUTINE SUMPRD(JMIN,JMAX
          EUVION(2,7,J)= EUVION(2,1,J)+EUVION(2,2,J)+EUVION(2,3,J)
          PEPION(2,7,J)= PEPION(2,1,J)+PEPION(2,2,J)+PEPION(2,3,J)
          PAUION(2,7,J)= PAUION(2,1,J)+PAUION(2,2,J)+PAUION(2,3,J)
+         if((mp.eq.17).and.(lp.eq.31).and.(j.eq.40))write(6,*)'yag ',

```

Comment:
this looks like a debug write, probably just want to revert this file and remove the mp,lp from SUMPRD calls?